typings/middleware/retry.d.ts

```diff
@@ -6,10 +6,22 @@ declare module 'mappersmith/middleware/retry' {
   export default Retry
 }
 
+type HttpMethod =
```

Comment:
For the endpoint definitions, lowercase method names are used. Should this be consistent?

Reply:
Done. Thanks Robin